a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -662,8 +667,8 @@ <h4>Display techniques for Visual adjustments support
 
 
 		<section id="supports-nonvisual-reading">
-			<h3 data-localization-id="nonvisual-reading-title">Support
-				for nonvisual reading</h3>
+			<h4 data-localization-id="nonvisual-reading-title">Support
```

Comment:
ID should be "ways-of-reading-nonvisual-reading-title"
to mirror what we did for "additional-accessibility-information-clarity-title"